packages/worker/src/index.ts

```diff
@@ -88,7 +86,6 @@ const shutdown = () => {
 export default server.listen(parseInt(env.PORT || "4002"), async () => {
   console.log(`Worker running on ${JSON.stringify(server.address())}`)
   await initPro()
-  await redis.init()
```

Comment:
Ah - this could be an issue, as mentioned earlier this will mean that there are no redis parameters being checked until a code is used - meaning that the worker could be mis-configured but it won't be flagged up until a code is needed.

Its sort of off topic, but it could be worth adding redis management to the worker, which purely calls through to init/shutdown commands in the backend-core redis parts that the worker needs - purely so that we can maintain similar functionality to what we have today.

Reply:
It's easy enough to re-instate the `init`/`shutdown` functions, there will just be 2 of them (one for invites, one for password reset). If that's no bother, I'll go do it.

Reply:
yeah - I think we probably should do that - expose the ability to init/shutdown just for feature parity.